### PR TITLE
Updated the documentation with json config examples

### PR DESCRIPTION
**Pull request by ravit-db**
- Added json config examples

### CODE REVIEW

File: docs/README_RECON.md [hunk 1]
```diff
@@ -72,9 +76,16 @@ flowchart TD
...
 
-```python
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:88]:
```suggestion
<pre lang="Python">
```

File: docs/README_RECON.md [hunk 2]
```diff
@@ -87,7 +98,28 @@ class Table:
     transformations: list[Transformation] | None = None
     thresholds: list[Thresholds] | None = None
     filters: Filters | None = None
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:104]:
```suggestion
<pre lang="JSON">
```

File: docs/README_RECON.md [hunk 3]
```diff
@@ -87,7 +98,28 @@ class Table:
...
+{
+  "source_name": "product",
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:106]:
```suggestion
"source_name": "<SOURCE_NAME>",
```

File: docs/README_RECON.md [hunk 4]
```diff
@@ -87,7 +98,28 @@ class Table:
...
+  "target_name": "product",
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:107]: Please update all the actual values with <TO-BE-FILLED> elements. Here we are just providing a reference on how you can update the structure.

File: docs/README_RECON.md [hunk 5]
```diff
@@ -104,15 +136,36 @@ class Table:
...
     upper_bound: str
     fetch_size: int = 100
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
+"jdbc_reader_options":{
+  "number_partitions": 10,
+  "partition_column": "p_id",
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:160]: Please update all the actual values with <TO-BE-FILLED> elements.

File: docs/README_RECON.md [hunk 6]
```diff
@@ -124,12 +177,33 @@ class JdbcReaderOptions:
...
 
-```python
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:187]: Same as above

File: docs/README_RECON.md [hunk 7]
```diff
@@ -124,12 +177,33 @@ class JdbcReaderOptions:
...
+
 @dataclass
 class ColumnMapping:
     source_name: str
     target_name: str
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:196]: same as above

File: docs/README_RECON.md [hunk 8]
```diff
@@ -124,12 +177,33 @@ class JdbcReaderOptions:
...
+"column_mapping":[
+  {
+    "source_name": "p_id",
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:199]: Please update all the actual values with <TO-BE-FILLED> elements.

File: docs/README_RECON.md [hunk 9]
```diff
@@ -138,30 +212,79 @@ class ColumnMapping:
...
 
-```python
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:222]: same as above

File: docs/README_RECON.md [hunk 10]
```diff
@@ -138,30 +212,79 @@ class ColumnMapping:
...
 class Transformation:
     column_name: str
     source: str
     target: str | None = None
-```
+
+</pre>
+</td>
+<td>
+<pre lang="json">
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:233]: same as above

File: docs/README_RECON.md [hunk 11]
```diff
@@ -138,30 +212,79 @@ class ColumnMapping:
...
+"transformations":[
+    {
+      "column_name": "product_price",
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:236]: Please update all the actual values with <TO-BE-FILLED> elements.

**Comment by ganeshdogiparthi-db** [comment]
Along with the JSON structure, we also need to add the pre-requisite required to run the tool in CLI format. It is confusing now, and there is a gap in documentation, I see. @ravit-db  @vijaypavann-db

File: docs/README_RECON.md [hunk 12]
```diff
@@ -87,7 +98,28 @@ class Table:
...
     filters: Filters | None = None
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
+{
+  "source_name": "product",
+  "target_name": "product",
+  "join_columns": ["p_id"],
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:108]: We can provide an example JSON file with all the details, at the end. (maybe collapsed one)

File: docs/README_RECON.md [hunk 13]
```diff
@@ -138,30 +212,79 @@ class ColumnMapping:
...
+## thresholds
+
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:262]: same as above

File: docs/README_RECON.md [hunk 14]
```diff
@@ -138,30 +212,79 @@ class ColumnMapping:
...
     type: str
-```
+
+</pre>
+</td>
+<td>
+<pre lang="json">
+"thresholds":[
+  {
+    "column_name": "p_price",
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:277]: Please update all the actual values with <TO-BE-FILLED> elements.

File: docs/README_RECON.md [hunk 15]
```diff
@@ -138,30 +212,79 @@ class ColumnMapping:
...
     column_name: str
     lower_bound: str
     upper_bound: str
     type: str
-```
+
+</pre>
+</td>
+<td>
+<pre lang="json">
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:274]: same as above

File: docs/README_RECON.md [hunk 16]
```diff
@@ -172,12 +295,31 @@ class Thresholds:
...
 class Filters:
     source: str | None = None
     target: str | None = None
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
+"filters":{
+  "source": "p_price > 1000",
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:315]: Please update all the actual values with <TO-BE-FILLED> elements.

File: docs/README_RECON.md [hunk 17]
```diff
@@ -172,12 +295,31 @@ class Thresholds:
...
 
-```python
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
```
Comment (vijaypavann-db) [inline docs/README_RECON.md:305]: same as above

**Review by vijaypavann-db — CHANGES_REQUESTED** [review]
Please update as per the comments

File: docs/README_RECON.md (hunk 1 above)
Comment (ravit-db) [reply docs/README_RECON.md:88]: Fixed

File: docs/README_RECON.md (hunk 2 above)
Comment (ravit-db) [reply docs/README_RECON.md:104]: Fixed

File: docs/README_RECON.md (hunk 3 above)
Comment (ravit-db) [reply docs/README_RECON.md:106]: Fixed

File: docs/README_RECON.md (hunk 5 above)
Comment (ravit-db) [reply docs/README_RECON.md:160]: Fixed

File: docs/README_RECON.md (hunk 6 above)
Comment (ravit-db) [reply docs/README_RECON.md:187]: Fixed

File: docs/README_RECON.md (hunk 7 above)
Comment (ravit-db) [reply docs/README_RECON.md:196]: Fixed

File: docs/README_RECON.md (hunk 8 above)
Comment (ravit-db) [reply docs/README_RECON.md:199]: Fixed

File: docs/README_RECON.md (hunk 9 above)
Comment (ravit-db) [reply docs/README_RECON.md:222]: Fixed

File: docs/README_RECON.md (hunk 10 above)
Comment (ravit-db) [reply docs/README_RECON.md:233]: Fixed

File: docs/README_RECON.md (hunk 11 above)
Comment (ravit-db) [reply docs/README_RECON.md:236]: Fixed

File: docs/README_RECON.md (hunk 12 above)
Comment (ravit-db) [reply docs/README_RECON.md:108]: Fixed

File: docs/README_RECON.md (hunk 13 above)
Comment (ravit-db) [reply docs/README_RECON.md:262]: Fixed

File: docs/README_RECON.md (hunk 14 above)
Comment (ravit-db) [reply docs/README_RECON.md:277]: Fixed

File: docs/README_RECON.md (hunk 15 above)
Comment (ravit-db) [reply docs/README_RECON.md:274]: Fixed

File: docs/README_RECON.md (hunk 16 above)
Comment (ravit-db) [reply docs/README_RECON.md:315]: Fixed

File: docs/README_RECON.md (hunk 17 above)
Comment (ravit-db) [reply docs/README_RECON.md:305]: Fixed

**Review by vijaypavann-db — APPROVED** [review]
LGTM

**Pull request merged**